src/components/CookieConsent/components/ConsentPopup/ConsentPopup.tsx

```diff
@@ -0,0 +1,268 @@
+import React from 'react';
+
+import {ArrowLeft, Xmark} from '@gravity-ui/icons';
+import {Button, Icon, Link, Modal, Text} from '@gravity-ui/uikit';
+
+import {block} from '../../../utils/cn';
+import {ConsentType} from '../../ConsentManager';
+import type {ConsentManager, Consents} from '../../ConsentManager';
+import {prepareConsent} from '../../helpers';
+import i18n from '../../i18n';
+import {FoldableList} from '../FoldableList/FoldableList';
+
+import {
+    ConsentPopupCookieListItem,
+    ConsentPopupProps,
+    ConsentPopupStep,
+    FooterProps,
+    HeaderProps,
+} from './types';
+
+import './ConsentPopup.scss';
+
+const b = block('consent-popup');
+
+const getCurrentConsents = (consentManager: ConsentManager) => {
```

Comment:
In consentManager we have getConsents that returns actual consents. If we haven't defined consetns yet,  we need to show consent component

But in ConsentPopup we need to define all consent types because we need to show them in FoldableList